Tools/GNUMake/comps/hip.mak

```diff
@@ -114,6 +115,15 @@ ifeq ($(HIP_COMPILER),clang)
   # rocThrust - Header only
   # SYSTEM_INCLUDE_LOCATIONS += $(ROC_PATH)/rocthrust/include
 
+  ifeq ($(USE_ROCTX),TRUE)
```

Comment:
We could also turn this on by default with GNUmake builds, as soon as HIP is found.